src/main/java/geekbrains/java2/ht1b/Main.java

```diff
@@ -0,0 +1,257 @@
+package geekbrains.java2.ht1b;
+
+import java.util.Scanner;
+
+public class Main {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.